Makefile

```diff
@@ -97,21 +100,27 @@ docker-build: ## Builds the launch container
 .PHONY: docker-test
 docker-test: validate-reqs docker-build ## Run the tests
 	@export AWS_DEFAULT_REGION="testing propagation to AWS_REGION var"; \
-	docker run                                                			\
-		-v "$(SIMULATOR_AWS_CREDS_PATH)":/home/launch/.aws  			\
-		--env-file launch-environment                       			\
-		--rm -t $(CONTAINER_NAME_LATEST) \
+	docker run                                                          \
+		-v "$(SIMULATOR_AWS_CREDS_PATH)":/home/launch/.aws          \
+		--env-file launch-environment                               \
+		--rm -t $(CONTAINER_NAME_LATEST)                            \
 		/app/test-acceptance.sh
 
 	cd attack && make docker-test
 
 # -- SIMULATOR CLI
 .PHONY: dep
 dep: ## Install dependencies for other targets
-	$(GO) mod download 2>&1
+	$(GO) mod download
+	$(GO) get honnef.co/go/tools/cmd/staticcheck
+
+.PHONY: static-analysis
+static-analysis: dep
+	$(GO) vet
+	staticcheck $(PKG)
```

Comment:
More of a comment rather than an issue:

This command runs the check against the default settings. I guess it would be worth checking if this will break with third-party dependencies.

Reply:
Both tools only run their checks against our code and not against the 3rd party deps